src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,72 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+
+    public int calculate(String inputString){
+        for(String splitData : inputString.split(" ")){
+            if(isSign(splitData)){
+                doArithmeticOperation(splitData);
+            } else {
+                numbers.push(Integer.parseInt(splitData));
+            }
```

Comment:
There is no should. "Should" do only in critical situations. one if-else inside of for is not critical so it's a question of taste to extract it or not